### PR TITLE
Make /play/practice/[deckId] URLs work

### PR DESCRIPTION
**Pull request by AlexNisnevich**
This is a prerequisite to #568.

`GameArea` unfortunately got a little messy because we need to load data from Firebase before we can get the cards in a deck, but it takes a few seconds to get a response back from Firebase on page load.

### CODE REVIEW

File: src/common/containers/GameArea.js [hunk 1]
```diff
@@ -298,9 +287,44 @@ export class GameArea extends Component {
     return this.props.player === 'blue' ? this.props.bluePieces : this.props.orangePieces;
   }
 
+  tryToStartGame(props = this.props) {
```
[the remark this answers is not in the document]
Comment (john-patterson) [reply src/common/containers/GameArea.js]: This should also get some form of test.

**Comment by AlexNisnevich** [comment]
@john-patterson How's it look now? (No tests yet)

**Comment by john-patterson** [comment]
@AlexNisnevich much better, lgtm pending tests.

**Comment by AlexNisnevich** [comment]
Got a test for you, @john-patterson !

File: test/containers/GameArea.spec.js [hunk 2]
```diff
@@ -122,4 +123,62 @@ describe('GameArea container', () => {
...
+    game = createGameArea(state, dispatch, {...historyParams, collection: {
+      firebaseLoaded: true,
+      decks: []
+    }});
+    renderElement(game, true);
+    expect(dispatchedActions.pop()).toEqual(
+      { type: 'HISTORY.PUSH', payload: { url: '/play' } }
+    );
+
+    // If collection.firebaseLoaded = true and the deck exists, start a practice game with that deck.
```
Comment (john-patterson) [inline test/containers/GameArea.spec.js]: These commented lines should be separate `it` blocks.

**Pull request merged**